src/components/v5/frame/NavigationSidebar/partials/NavigationSidebarButton/NavigationSidebarButton.tsx

```diff
@@ -13,17 +13,23 @@ const NavigationSidebarButton: FC<NavigationSidebarButtonProps> = ({
   iconName,
   label,
   className,
+  secondLevelMenuProps,
+  hideMobile,
   isActive,
   isExpanded,
   ...rest
 }) => {
   const isTablet = useTablet();
 
+  // Don't show item if hideMobile is true and we're on mobile
+  if (isTablet && hideMobile) {
```

Comment:
You should hide the whole item (li), and not just the button. I think that this prop should be moved to a different component